src/includes_normalize-win32.cc

```diff
@@ -96,8 +96,12 @@ string IncludesNormalize::Relativize(StringPiece path, const string& start) {
 
 string IncludesNormalize::Normalize(const string& input,
                                     const char* relative_to) {
-  char copy[_MAX_PATH];
+  char copy[_MAX_PATH + 1];
   size_t len = input.size();
+  if (len > _MAX_PATH) {
```

Comment:
Do you mean using >= in the comparison? I think it's correct as is, I made the buffer _MAX_PATH + 1 so that it can handle _MAX_PATH-lengthed paths which is as large as real paths should be.